CHANGELOG.md

```diff
@@ -0,0 +1,51 @@
+# Changelog
+
+This changelog contains changes starting from version v0.1.0
+
+# Version 0.2.0
+
+## Safe{Core} Protocol contracts
+
+### Changes
+
+- Issue [#47](https://github.com/5afe/safe-core-protocol/issues/47)
+
+    Allow setting Function handler for individual function selector per safe
+
+- Issues [#73](https://github.com/5afe/safe-core-protocol/issues/73)
+
+    Check if contract supports appropriate interface in enabling as Plugin
+
+- Issue [#46](https://github.com/5afe/safe-core-protocol/issues/46)
+    
+    Add support for hooks in multi-signature execution flow for a Safe
+
+- Issue [#62](https://github.com/5afe/safe-core-protocol/issues/62)
+
+    Allow `to` value to be account address in executeTransaction when caller has root access #62 
+
+- Issue [#71](https://github.com/5afe/safe-core-protocol/issues/71)
+    
+    Fixed broken linked list after disabling plugin in Manager
```

Comment:
please run these sentences through Grammarly to check if they're grammatically correct in English